modules/cactus-web/src/Tooltip/Tooltip.tsx

```diff
@@ -46,54 +50,75 @@ const cactusPosition: Position = (triggerRect, tooltipRect) => {
   }
 
   const directionRight = collisions.left && !collisions.right
-  const directionUp = collisions.bottom && !collisions.top
+  const directionBottom = collisions.top && !collisions.bottom
 
-  if (directionRight && !directionUp) {
-    styles.borderTopLeftRadius = '0px'
-  } else if (directionRight && directionUp) {
+  if (directionRight && !directionBottom) {
     styles.borderBottomLeftRadius = '0px'
-  } else if (!directionRight && !directionUp) {
-    styles.borderTopRightRadius = '0px'
-  } else if (!directionRight && directionUp) {
+  } else if (directionRight && directionBottom) {
+    styles.borderTopLeftRadius = '0px'
+  } else if (!directionRight && !directionBottom) {
     styles.borderBottomRightRadius = '0px'
+  } else if (!directionRight && directionBottom) {
+    styles.borderTopRightRadius = '0px'
   }
 
   return {
     ...styles,
     left: directionRight
       ? triggerRect.left + OFFSET + scrollX
       : triggerRect.right - OFFSET - tooltipRect.width + scrollX,
-    top: directionUp
-      ? triggerRect.top - tooltipRect.height + scrollY
-      : triggerRect.top + triggerRect.height + scrollY,
+    top: directionBottom
+      ? triggerRect.top + triggerRect.height + scrollY
+      : triggerRect.top - tooltipRect.height + scrollY,
     // setting width to itself explicitly prevents "drift"
     width: tooltipRect.width,
   }
 }
 
 interface StyledInfoProps {
   disabled?: boolean
+  forceVisible?: boolean
 }
 
+const getStyledInfoColor = (props: StyledInfoProps): ReturnType<typeof css> => {
+  if (props.disabled) {
+    return css`
+      color: ${(p): string => p.theme.colors.mediumGray};
+    `
+  } else if (props.forceVisible) {
+    return css`
+      color: ${(p): string => p.theme.colors.callToAction};
+    `
+  }
+  return css`
+    color: ${(p): string => p.theme.colors.darkestContrast};
+  `
+}
 const StyledInfo = styled(NotificationInfo)<StyledInfoProps>`
```

Comment:
One more thing.  I'm seeing console errors related to the fact that the `forceVisible` prop is being passed all the way down to the icon SVG, and React isn't recognizing it as a valid HTML prop.  We'll need to filter it out to get rid of that error.

```suggestion
styled(({ forceVisible, ...props }) => <NotificationInfo {...props} />)<
  StyledInfoProps
>`
```

Reply:
Alternative ways of doing the same thing: https://styled-components.com/docs/api#transient-props

Reply:
I liked Glen's idea. Updated

Reply:
One downside to transient props, however, is that it's only available in Styled Components v5.  That would require a breaking Cactus change, as currently we support apps that use Styled Components v4.  Therefore my vote is to do it the non-transient way, as I don't think this refactor is worth a breaking change.